Mathlib/Algebra/Group/Defs.lean

```diff
@@ -590,6 +590,8 @@ lemma pow_mul_comm' (a : M) (n : ℕ) : a ^ n * a = a * a ^ n := by rw [← pow_
 -- TODO: Should `alias` automatically transfer `to_additive` statements?
 @[to_additive existing two_nsmul] alias sq := pow_two
 
+lemma mul_self (a : M) : a * a = a ^ 2 := (sq a).symm
```

Comment:
```suggestion
@[to_additive add_self_eq_two_nsmul]
lemma mul_self (a : M) : a * a = a ^ 2 := (sq a).symm
```
otherwise people will get confusing error messages when using this lemma